src/mk.sh

```diff
@@ -127,7 +111,15 @@ function modules_install
 
   case "$target" in
     1) # VM_TARGET
-      vm_modules_install
+      local distro=$(detect_distro "${configurations[mount_point]}/")
+
+      if [[ "$distro" != "debian" ]]; then
+        complain "Unfortunately, there's no support for the target distro"
+        vm_umount
+        exit 95 # ENOTSUP
+      fi
+
+      modules_install_to "${configurations[mount_point]}"
```

Comment:
I did this from our discussion of aborting the process as soon as I know that it cannot be completed. That is, to cancel early in the process and not in the middle of it.  The previous behavior was to install modules and, after that, check if the distro is valid, aborting the deployment in the middle of the "bd --vm" execution. However, I will work on the arch version, and, probably, it will not be necessary.